Readme.md

```diff
@@ -191,6 +192,21 @@ The timeout value (in milliseconds) for the worker to stop before sending SIGKIL
 
 **Kind**: instance property of [<code>WorkerNodesOptions</code>](#WorkerNodesOptions)  
 **Default**: <code>100</code>  
+<a name="WorkerNodesOptions+resourceLimits"></a>
+
+### options.resourceLimits : <code>Object</code>
+Provides the set of JS engine resource constraints inside this Worker thread.
+
+**Kind**: instance property of [<code>WorkerNodesOptions</code>](#WorkerNodesOptions)  
+**Properties**
+
+| Name | Type | Description |
+| --- | --- | --- |
+| maxYoungGenerationSizeMb | <code>Number</code> | The maximum size of the main heap in MB |
```

Comment:
Descriptions should be reversed: maxYoung - recently created objects, maxOld - "main heap".

Reply:
done ;)